web-animations-1/Overview.bs

```diff
@@ -4877,6 +4877,22 @@ The {{Animation}} interface {#the-animation-interface}
       calling this method <em>does</em> trigger a [=style change event=]
       (see [[#model-liveness]]).
 
+      Furthermore,
```

Comment:
For this to work, do we not also have to require that both style flushes during commitStyles are part of the _same_ style change event? Otherwise, transitions will be generated by the second flush?